sonarqube-build.gradle.kts

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:4.1.0.3113")
+        classpath("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:4.2.0.3127")
```

Comment:
wait before merging to clarify with release engineers what version is better to have: `4.2.0.3127` or `4.2.0.3129`